### ISSUE TITLE
HELP SHOW LOAD doesn't say SHOW LOAD cannot be used for stream  load

### ISSUE THREAD

**Issue by creatstar** [apache/doris#4077]
**Describe the bug**
HELP SHOW LOAD doesn't say SHOW LOAD cannot be used for stream  load.

**To Reproduce**
1. help show load

**Expected behavior**
The stream load tasks are not showed in SHOW LOAD command. This content should be included in "HELP SHOW LOAD".

**Comment by yangzhg** [comment]
Stream load cannot show  in `SHOW LOAD` statment , because of  stream load is a synchronized http api

**Comment by creatstar** [comment]
Yes, SHOW LOAD doesn't support Stream Load should be indicated in the Help Text. Otherwise, the reader would not get that point.
